yarn.lock

```diff
@@ -5866,7 +5866,25 @@ node-pre-gyp@^0.6.29, node-pre-gyp@~0.6.32:
     tar "^2.2.1"
     tar-pack "^3.4.0"
 
-node-sass@^3.10.1, node-sass@^3.4:
+node-sass@3.4.2:
```

Comment:
It looks like USWDS requires "node-sass": "^3.4.2", but we are currently requiring "^3.10.1", and installing "3.13.1". This PR requires the version of node-sass that USWDS requires. Is it better to simply upgrade to the latest version?